Reflection/Interfaces.simba

```diff
@@ -435,117 +270,13 @@ begin
 
-(*
-LOGOUT INTERFACE
-~~~~~~~~~~~~~~~~
-
-  Anything in the Logout Tab.
-*)
-Function R_Logout: Boolean;
-var
-  Widget: RSWidget;
-begin
-  if (not R_IsGametabOpen(GAMETAB_LOGOUT)) then
-    R_OpenGametab(GAMETAB_LOGOUT);
-  Wait(50, RandomRange(250, 692));
-
-  if (RSWidget.IsValid(R_LOGOUT_CONTAINER.Group, R_LOGOUT_CONTAINER.Child)) and (not R_WidgetIsHidden(R_LOGOUT_LOGOUT_BUTTON)) then
-  begin
-    Widget := RSWidget.Get(R_LOGOUT_LOGOUT_BUTTON.Group, R_LOGOUT_LOGOUT_BUTTON.Child);
-    if Widget.ref <> nil then
-    begin
-      Mouse.Click(Widget.Bounds, MOUSE_LEFT);
-      Result := True;
-      Widget.Free;
-    end;
-  end else if (RSWidget.IsValid(R_WORLD_SWITCHER_CONTAINER.Group, R_WORLD_SWITCHER_CONTAINER.Child)) and (not R_WidgetIsHidden(R_WORLD_SWITCHER_LOGOUT_BUTTON)) then
-  begin
-    Widget := RSWidget.Get(R_WORLD_SWITCHER_LOGOUT_BUTTON.Group, R_WORLD_SWITCHER_LOGOUT_BUTTON.Child);
-    if Widget.ref <> nil then
-    begin
-      Mouse.Click(Widget.Bounds, MOUSE_LEFT);
-      Result := True;
-      Widget.Free;
-    end;
-  end;
-end;
-
-Function R_CloseWorldSwitcher(): Boolean;
-var
-  Widget: RSWidget;
-begin
-  if (not R_IsGametabOpen(GAMETAB_LOGOUT)) then
-    R_OpenGametab(GAMETAB_LOGOUT);
-  Wait(50, RandomRange(250, 692));
-
-  if (RSWidget.IsValid(R_WORLD_SWITCHER_CONTAINER.Group, R_WORLD_SWITCHER_CONTAINER.Child)) and (not R_WidgetIsHidden(R_WORLD_SWITCHER_CLOSE)) then
-  begin
-    Widget := RSWidget.Get(R_WORLD_SWITCHER_CLOSE.Group, R_WORLD_SWITCHER_CLOSE.Child);
-    if Widget.ref <> nil then
-    begin
-      Mouse.Click(Widget.Bounds, MOUSE_LEFT);
-      Result := True;
-      Widget.Free;
-    end;
-  end
-
-end;
-
-Function R_OpenWorldSwitcher: Boolean;
-var
-  Widget: RSWidget;
-begin
-  if (not R_IsGametabOpen(GAMETAB_LOGOUT)) then
-    R_OpenGametab(GAMETAB_LOGOUT);
-  Wait(50, RandomRange(250, 692));
-  if (not RSWidget.IsValid(R_LOGOUT_CONTAINER.Group, R_LOGOUT_CONTAINER.Child)) or R_WidgetIsHidden(R_LOGOUT_WORLD_SWITCHER_BUTTON) then
-    Exit(False);
-
-  Widget := RSWidget.Get(R_LOGOUT_WORLD_SWITCHER_BUTTON.Group, R_LOGOUT_WORLD_SWITCHER_BUTTON.Child);
-  if Widget.ref <> nil then
-  begin
-    Mouse.Click(Widget.Bounds, MOUSE_LEFT);
-    Result := True;
-    Widget.Free;
-  end;
-end;
-
 (*
 MAGIC SPELL BOOK
 ~~~~~~~~~~~~~~~~
 
   Functions for using the spell book.
 *)
 
-Function R_SpellBook: Boolean;
-begin
-  Result := RSWidget.IsValid(R_SPELLBOOK_CONTAINER.Group, R_SPELLBOOK_CONTAINER.Child);
-end;
-
-Function R_WaitSpellBook(timeout: UInt32): Boolean;
-begin
-  Result := WaitUntil(R_SpellBook, 50, timeout);
-end;
-
-Function R_CastSpell(Spell: Int32): Boolean;
-var
-  ID: RWidget;
-  Widget: RSWidget;
-begin
-  ID := R_SpellIDs[Spell];
-  if (not R_SpellBook) or R_WidgetIsHidden(ID) then
-    Exit(False);
-
-  Widget := RSWidget.Get(ID.Group, ID.Child);
-  if Widget.ref <> nil then
-  begin
-    Mouse.Click(Widget.Bounds, MOUSE_LEFT);
-    Result := True;
-    Widget.Free;
-  end;
-end;
-
-
```

Comment:
Same with this, why are we deleting it? I don't see alternatives added anywhere else in this PR

Reply:
Same as above.